common/culture/innovations/00_early_medieval_innovations.txt

```diff
@@ -222,4 +224,29 @@ innovation_baliffs = {
 	flag = early_medieval_era_regular
 }
 
-#culture_group_regional
+#Warcraft
+innovation_animal_war_training = {
```

Comment:
Not going to lie, I don't see a big reason to add it. There's no design behind it.

I wouldn't touch vanilla techs if we don't aim at making the experience better. For me, it just buffs 4 random unit types that were doing well even before.

Reply:
This was basically the outcome of the question/issue from here: https://discord.com/channels/317351018050355200/751109648869752954/835463641415024690.
Didn't sound like you wouldn't like the idea...